meta-aos-tune/recipes-aos/aos-communicationmanager/aos-communicationmanager_git.bbappend

```diff
@@ -0,0 +1,27 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+SRC_URI_append = " \
```

Comment:
Most of this recipe is in meta-aos. I guess this recipe should look like:
```sh
FILESEXTRAPATHS_prepend := "${THISDIR}/files:"

pkg_postinst_${PN}() {
    # Add aoscm to /etc/hosts
    if ! grep -q 'aoscm' $D${sysconfdir}/hosts ; then
        echo '127.0.0.1	aoscm' >> $D${sysconfdir}/hosts
    fi
}
```

Reply:
no, config file differs from the orginal